web/packages/teleport/src/LocksV2/NewLock/LockCheckout/LockCheckout.tsx

```diff
@@ -0,0 +1,369 @@
+  }
+
+  function deleteLock(resource: LockResource) {
+    setAttempt({ status: '' });
+    toggleResource(resource);
+  }
+
+  const submitBtnDisabled =
+    locks.length === 0 || attempt.status === 'processing';
+
+  // Listeners are attached to enable overflow on the parent container after
+  // transitioning ends (entered) or starts (exits). Enables vertical scrolling
+  // when content gets too big.
+  //
+  // Overflow is initially hidden to prevent
+  // brief flashing of horizontal scroll bar resulting from positioning
+  // the container off screen to the right for the slide affect.
+  useEffect(() => {
+    function applyOverflowAutoStyle(e: TransitionEvent) {
+      if (e.propertyName === 'right') {
+        sliderRef.current.style.overflow = `auto`;
+        // There will only ever be one 'end right' transition invoked event, so we remove it
+        // afterwards, and listen for the 'start right' transition which is only invoked
+        // when user exits this component.
+        window.removeEventListener('transitionend', applyOverflowAutoStyle);
+        window.addEventListener('transitionstart', applyOverflowHiddenStyle);
+      }
+    }
+
+    function applyOverflowHiddenStyle(e: TransitionEvent) {
+      if (e.propertyName === 'right') {
+        sliderRef.current.style.overflow = `hidden`;
+      }
+    }
+
+    window.addEventListener('transitionend', applyOverflowAutoStyle);
+
+    return () => {
+      window.removeEventListener('transitionend', applyOverflowAutoStyle);
+      window.removeEventListener('transitionstart', applyOverflowHiddenStyle);
+    };
+  }, []);
+
+  return (
+    <Box
+      ref={sliderRef}
+      css={`
+        position: absolute;
+        width: 100vw;
+        height: 100vh;
+        top: 0;
+        left: 0;
+        overflow: hidden;
+      `}
+    >
+      <Dimmer className={transitionState} />
+      <SidePanel state={transitionState} className={transitionState}>
+        {attempt.status === 'success' ? (
+          <Box>
+            <Box mt={2} mb={7} textAlign="center">
+              <Text typography="h4" color="text.primary" bold>
+                Resources Locked Successfully
+              </Text>
+              <Text typography="subtitle1" color="text.secondary">
+                You've successfully locked {createdLocks.length}{' '}
+                {pluralize(createdLocks.length, 'resource')}
+              </Text>
+            </Box>
+            <Flex justifyContent="center" mb={3}>
+              <Image src={shieldCheck} width="250px" height="179px" />
+            </Flex>
+          </Box>
+        ) : (
+          <Flex mb={3} alignItems="center">
+            <ArrowBack
+              fontSize={25}
+              mr={3}
+              onClick={onClose}
+              style={{ cursor: 'pointer' }}
+            />
+            <Box>
+              <Text typography="h4" color="text.primary" bold>
+                {locks.length} {pluralize(locks.length, 'Resource')} Selected
```

Comment:
```suggestion
                {locks.length} {pluralize(locks.length, 'Target')} Added
```

Again, not 100% sold here as you _are_ selecting resources TO lock so you can skip this one.